### ISSUE TITLE
 --version

### ISSUE THREAD

**Issue by shafiuzzaman-md** [niklasso/minisat#46]

**Comment by jwaldmann** [comment]
yes. it'd be nice if minisat could print its version. it's in the source but inconsistently: https://github.com/niklasso/minisat/search?q=version
```
/CMakeLists.txt
...
set(MINISAT_SOMAJOR   2)
set(MINISAT_SOMINOR   1)    <=== !?
set(MINISAT_SORELEASE 0)

doc/ReleaseNotes-2.2.0.txt
```

And, Fedora claims to have 2.2.1 https://src.fedoraproject.org/rpms/minisat2 I wonder where they got that from. The diff is substantial (from this repo here, to that unpacked srpm)
